### PR TITLE
feat: expert reports (FPLA: 1410)

### PR DESCRIPTION
**Pull request by toireasam**
description:
adding expert report event

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-1410

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -978,5 +978,68 @@
     "CaseEventID": "addCaseNote",
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
+  },
+  {
```
Comment (swalker125) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json:982]: I would suggest making this available in all states. Court staff do cannot see a case until it's in submitted onwards anyway

Comment (toireasam) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json:982]: cool will do this :) i've wrote this down as a reminder court stall can't see until submitted. haha!!

File: ccd-definition/FixedLists/ExpertReportList.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    "DisplayOrder": 22
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "professionalOther",
+    "ListElement": "Professional: Other",
+    "DisplayOrder": 23
+  }
+]
```
Comment (swalker125) [inline ccd-definition/FixedLists/ExpertReportList.json:163]: are you sure this definiton uploads? I seem to remember a 40 character limit on list elements in fixed lists 🤔

Comment (toireasam) [reply ccd-definition/FixedLists/ExpertReportList.json:163]: uploads fine as we discussed. :)

**Pull request merged**